docker/build/Dockerfile.aarch64

```diff
@@ -17,5 +17,11 @@ RUN /build-libusb-and-hidapi.sh
 COPY ./build/build-rust.sh /build-rust.sh
 RUN /build-rust.sh
 ARG OCTEZ_VERSION
+RUN git clone --single-branch --depth 1 --branch "$OCTEZ_VERSION" https://gitlab.com/tezos/tezos.git
+WORKDIR /tezos
+ENV BLST_PORTABLE yes
+ENV OPAMYES true
+COPY ./build/build-deps.sh /build-deps.sh
+RUN /build-deps.sh
 COPY ./build/build-tezos.sh /build-tezos.sh
-RUN /build-tezos.sh ${OCTEZ_VERSION}
+RUN /build-tezos.sh
```

Comment:
:thinking: I'm not a docker expert, but does this change mean that in case there is a failure in `build-tezos.sh` what was done by `build-deps.sh` won't be repeated in a new run?

Reply:
`RUN` command creates intermediary image (layer) which gets cached

Reply:
exactly that change is unrelated, since `$OCTEZ_VERSION` is build time arg it won't prevent caching in any way

Reply:
but octez version is irrelevant for script now